src/types.jl

```diff
@@ -284,23 +306,25 @@ trialstep!{O,S}(::Solver{O,S}, ::AbstractState) =
 
 """
 
-Accepts (in-place) the computed step if `errorcontrol!` gave a small
-enough error.
+Estimates the error (such that a step is accepted if err<=1).
+Depending on the stepper it may update the state, e.g. by computing a
+new dt or a new order (but not by computing a new solution!).
 
-Returns `Status`.
+Returns `(err,Status)`.
 
+Defaults to return (0,cont), i.e. accept step.  This can be used for
+fixed-step solvers where no error control is done.
 """
-accept!{O,S}(::Solver{O,S}, ::AbstractState) =
-    error("Function `accept!` and companions (or alternatively `onestep!`) need to be implemented for adaptive solver $S")
+errorcontrol!{T}(::Solver,::AbstractState{T}) = zero(T), cont
```

Comment:
I'm not sure if this is necessary, I would rather have a fixed stepper implement the whole `onestep!`.  This way you don't have to implement `accept!` (which doesn't make sense for a fixed stepper).